### PR TITLE
Added Documentation Folder and About Us Page

### PR DESCRIPTION
**Pull request by Calebux**
I created a draft Docs folder with Docusaurus 
I added Documentation gotten from the Joyboy github repo
Design new about page on the Joyboy community website
Added a draft "about us" page  gotten from Abdel's description and joyboy's official twitter handle

### CODE REVIEW

**Comment by Calebux** [comment]
Screenshot 2024-05-29 at 14 57 46
Screenshot 2024-05-29 at 14 57 51
Screenshot 2024-05-29 at 14 58 13
Screenshot 2024-05-29 at 14 58 20
Screenshot 2024-05-29 at 14 58 29
Docs created with Docusaurus

**Comment by Calebux** [comment]
Screenshot 2024-05-29 at 15 00 01
Preview of About us

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
+// @ts-check
+// `@type` JSDoc annotations allow editor autocompletion and type checking
+// (when paired with `@ts-check`).
+// There are various equivalent ways to declare your Docusaurus config.
+// See: https://docusaurus.io/docs/api/docusaurus-config
+
+import { themes as prismThemes } from "prism-react-renderer";
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
```
Comment (ugur-eren) [inline docs/docusaurus.config.js:10]: Config should be customized to fit the Joyboy, can you change necessary fields?

File: docs/src/pages/markdown-page.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (ugur-eren) [inline docs/src/pages/markdown-page.md:1]: Example pages should be removed

**Review by ugur-eren — CHANGES_REQUESTED** [review]
Thanks for the work! I've requested some small changes. Also, can you replace the icons, logos etc with Joyboy images?

File: docs/src/components/HomepageFeatures/index.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
+import clsx from 'clsx';
```
[the remark this answers is not in the document]
Comment (Calebux) [reply docs/src/components/HomepageFeatures/index.js:1]: It is been used in line 40  - https://github.com/Calebux/joyboy/blob/44a2a29358b6079e7422035b1117c37f9fff9030/docs/src/components/HomepageFeatures/index.js#L40

[the remark this answers is not in the document]
Comment (ugur-eren) [reply docs/src/components/HomepageFeatures/index.js:1]: However `HomepageFeatures` is not used anywhere, it is imported here: https://github.com/Calebux/joyboy/blob/44a2a29358b6079e7422035b1117c37f9fff9030/docs/src/pages/index.js#L5
But no usage

File: docs/docusaurus.config.js (hunk 1 above)
Comment (ugur-eren) [reply docs/docusaurus.config.js:10]: There are still some fields that must changed in the config.
- copyright
- url (https://docs.joyboycommunity for now)
- tagline
- organizationName should be `keep-starknet-strange`
- docs & blog editUrl
- logo alt

**Review by ugur-eren — APPROVED** [review]
Thanks for the work!

**Pull request merged**